CODEOWNERS

```diff
@@ -1,40 +1,40 @@
-*                                                   @awslabs/rust-sdk-owners
+*                                                   @smithy-lang/aws-sdk-rust
 
 # Server
-/codegen-server-test/                               @awslabs/smithy-rs-server
-/codegen-server/                                    @awslabs/smithy-rs-server
-/rust-runtime/aws-smithy-http-server/               @awslabs/smithy-rs-server
+/codegen-server-test/                               @smithy-lang/smithy-rs-server
+/codegen-server/                                    @smithy-lang/smithy-rs-server
+/rust-runtime/aws-smithy-http-server/               @smithy-lang/smithy-rs-server
 
 # Python Server
-/codegen-server-test/python/                        @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
-/codegen-server/python/                             @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
-/rust-runtime/aws-smithy-http-server-python/        @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
+/codegen-server-test/python/                        @smithy-lang/smithy-rs-python-server @smithy-lang/smithy-rs-server
+/codegen-server/python/                             @smithy-lang/smithy-rs-python-server @smithy-lang/smithy-rs-server
+/rust-runtime/aws-smithy-http-server-python/        @smithy-lang/smithy-rs-python-server @smithy-lang/smithy-rs-server
 
 # Typescript Server
-/codegen-server-test/typescript/                    @awslabs/smithy-rs-typescript-server @awslabs/smithy-rs-server
-/codegen-server/typescript/                         @awslabs/smithy-rs-typescript-server @awslabs/smithy-rs-server
-/rust-runtime/aws-smithy-http-server-typescript/    @awslabs/smithy-rs-typescript-server @awslabs/smithy-rs-server
+/codegen-server-test/typescript/                    @smithy-lang/smithy-rs-typescript-server @smithy-lang/smithy-rs-server
+/codegen-server/typescript/                         @smithy-lang/smithy-rs-typescript-server @smithy-lang/smithy-rs-server
+/rust-runtime/aws-smithy-http-server-typescript/    @smithy-lang/smithy-rs-typescript-server @smithy-lang/smithy-rs-server
```

Comment:
@drganjoo, @david-perez 
I don't see `smithy-rs-typescript-server` or `smithy-rs-python-server` under [smithy-lang](https://github.com/orgs/smithy-lang/teams/) but do see them under [awslabs](https://github.com/orgs/awslabs/teams). Do they need to be moved to `smithy-lang`?

Reply:
if we want them to continue existing, we do. Not clear on that, cc @drganjoo

Reply:
Talked to @drganjoo. We'll change these ones to `smithy-rs-server`.